themes/kiali/static/css/kiali.css

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css?family=Poppins:300,400,500&display=swap');
+@import url("https://fonts.googleapis.com/css?family=Fira+Sans+Condensed&display=swap");
```

Comment:
Please, specify the '300,400,500' weights, and add the other weights you used.
There are some bolder and thinners texts that are lost.

For example the text "Kiali answers the questions:" is no longer rendering bolder than the listed questions.